core/backend/src/CloudSqliteMock.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import * as sinon from "sinon";
+import { CloudSqlite } from "./CloudSqlite";
+import { HubMock } from "./HubMock";
+import { CheckpointProps } from "./CheckpointManager";
+
+/**
+ * Mocks the CloudSqlite access needed for [[V2CheckpointManager.downloadCheckpoint]].
+ * @internal
+ */
+export class CloudSqliteMock {
```

Comment:
i would move this class into the test dir since its a util, and we dont necessarily want to expose this as part of the pkg api

Reply:
I put it next to HubMock.ts, which is used in the same way. I could move both files, but even though HubMock.ts doesn't use sinon, I do feel that both classes go together.

Reply:
To be honest, I feel that HubMock.ts should probably be moved to the test dir, but I have left that alone for now. Now that CloudSqliteMock is only accessed from HubMock, I moved CloudSqliteMock.ts into the test directory.